.github/actions/add-jira-issue-code-to-pull-request/Add-JiraIssueCodeToPullRequest.ps1

```diff
@@ -41,6 +41,11 @@ elseif ($Body -NotLike "*``[$issueKey``]``($jiraBrowseUrl$issuekey``)*")
 
 if (($Title -ne $originalTitle) -or ($Body -ne $originalBody))
 {
+    # Escape the quote characters.
+    $Title = $Title -replace '"','\"'
+    $Body = $Body -replace '"','\"'
```

Comment:
Down at the `gh` call the inputs are not part of the code, right? So, one can't do script injection via these parameters?

Reply:
I've added both `$(...)` and `` `...` `` in the title. The nothing changed after the subsequent run. If the title was evaluated in `pwsh` then it would've thrown an error on the `` `...` `` and if it was evaluated in `sh` then it would've thrown an error on the `$(...)`. If it was evaluated in bash or zsh then the title would've changed after the run. So we are good to go.

I did some research and it seems like a Powershell-specific problem. If I understand correctly it removes the quotes when an argument is passed from the PS runtime into a native command like the GH CLI. (see https://github.com/cli/cli/issues/3425)

Reply:
I see, good then. Please include this explanation in the code too.

Reply:
ok, [I've added a comment](https://github.com/Lombiq/GitHub-Actions/pull/183/commits/810c01383a168f5f17aa6827751d734260e3ffa9)